src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -379,6 +383,32 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
       stay() using data.copy(startMode = Option(startMode))
   }
 
+  private def handleCallPreempted(callKey: OutputKey,
```

Comment:
Preempted might not be the right terminology outside of the jes backend. Other similar things might call it something else.